runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ExternalPythonExecutor.java

```diff
@@ -100,7 +100,7 @@ public PythonEvaluationResult eval(String expression, String prepareEnvironmentS
         try {
             String pythonPath = checkPythonPath();
             tempEvalEnvironment = generateTempResourcesForEval();
-            String payload = generatePayloadForEval(expression.replace("\n", ""), prepareEnvironmentScript, context);
```

Comment:
could move the code jnside generatePayloadForEval
Could make use of StringUtils.replaceChars

Reply:
Done.